examples/regexp-type/regexp_type.md

```diff
@@ -0,0 +1,16 @@
+# RegExp 
+
+`RegExp` is a built-in subtype of `anydata`.
+
+The lang.regexp module contains various regular expression related functions. You can learn more about these functions and how to use them by referring to the lang.regexp API documentation.
```

Comment:
```suggestion
The `lang.regexp` module contains various functions related to regular expressions. You can learn more about these functions and how to use them by referring to the `lang.regexp` API documentation.
```
Should we add the API Docs link?

Reply:
We can add it in the Related Links section. Ryt?